lib/index.js

```diff
@@ -523,6 +532,36 @@ export default {
 		this.toolbar.update({ buildInProgress: true });
 	},
 
+	/**
+	 * Take a screenshot of a device/simulator/emulator
+	 */
+	takeScreenshot() {
+		const filename = atom.config.get('appcelerator-titanium.general.screenshotPath') + "screenshot_" + new Date().getTime() + '.png';
+		const exec = require('child_process').exec;
+
+		if (os.platform() === 'darwin') {
```

Comment:
Is this enough to determine if a Simulator is currently running? Some examples where this **could** fail:
- No Simulator running
- Build running for device, ad-hoc or production
- Running an Android / Windows build on a Mac

Reply:
@m1ga Should we help here? Let us know!

Reply:
I think to check for all this would be some more work. E.g. on Android you could list the devices and check if there are multiple instances and use them in the screenshot command. On iOS I'm not sure how to check it. The current implementation would just not take a screenshot if the command doesn't exit with a success and display an error message.

Any help is of course very appreciated :smile: